.github/workflows/ci.yml

```diff
@@ -160,6 +174,10 @@ jobs:
         with:
           path: /home/runner/.cache/Cypress
           key: ${{ runner.os }}-cypress-${{secrets.ACTIONS_CACHE_VERSION}}-${{ hashFiles('yarn.lock') }}
+      - name: generate prisma
+        run: cd typescript/db && yarn deploy && yarn generate
+        env:
+          POSTGRES_EXTERNAL_URL: ${{secrets.POSTGRES_EXTERNAL_URL}}
```

Comment:
Why do we use a secret here in the e2e tests ? It should be hardcoded to use the local db, not the production db

Reply:
Actually in our secrets in Github `POSTGRES_EXTERNAL_URL` is pointing to localhost, so this is indeed the local DB, should I change it to a hardcoded one?

Reply:
Strange... It shouldn't, it should be the Supabase URL. So yes here, localhost is not a secret, it should just be hardcoded for the test.

Reply:
Ok will replace all references to that secret in the CI file by the hardcoded localhost host (it's set as well for the unit tests)

Reply:
Done in 61cba8ee89c11c9cd29d46d835ec56359033e7d3